.github/stale.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+# daysUntilClose: 7
```

Comment:
Leaving this commented-out means that the default value is used, which is 7.

Reply:
I would try to set it either false or then to a ridiculously big number 2^32 perhaps.

Reply:
I think `false` will break the script: https://github.com/probot/stale/blob/master/lib/stale.js#L39

Reply:
So we should rather go for a big number. And perhaps file a feature request for the bot developers.

Reply:
I'll handle the reopen of all PRs once that is changed.

Reply:
Feature request: https://github.com/probot/stale/issues/79